src/styles/blocks/subscripe.scss

```diff
@@ -0,0 +1,136 @@
+.subscripte {
+  background-image: url(../images/photos/8.png);
+  height: 386px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  @include onTablet {
+    height: 380px;
+  }
+
+  @include onDesktop {
+    height: 550px;
+  }
+
+  &__title {
+    font-family: "Montserrat", sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 36px;
+    line-height: 36px;
+    text-align: center;
+    color: #fff;
+
+    @include onTablet {
+      font-size: 48px;
+      line-height: 48px;
+    }
+
+    @include onDesktop {
+      font-size: 48px;
+      line-height: 48px;
+    }
+  }
+
+  &__descr {
+    font-family: "IBM Plex Sans", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 18px;
+    line-height: 27px;
+    text-align: center;
+    color: #fff;
+
+    @include onTablet {
+      line-height: 18px;
+    }
+
+    @include onDesktop {
+      line-height: 18px;
+    }
+  }
+
+  &__button {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+  }
+
+  &__field {
+    box-sizing: border-box;
+    display: block;
+    width: 230px;
+    height: 50px;
+    padding: 14px 16px;
+    background: #f8f8fa;
+    border: 1px solid transparent;
+    font-family: "Montserrat", sans-serif;
+
+    &::placeholder {
+      line-height: 20px;
+      color: #7e7e83;
+    }
+
+    &:hover {
+      border-color: #d5d7de;
+      color: #161616;
+    }
+
+    &:focus {
+      border-color: #161616;
+      color: #161616;
+      outline: none;
+    }
+
+    @include onTablet {
+      margin-right: 16px;
+      width: 374px;
+      height: 70px;
+    }
+
+    @include onDesktop {
+      margin-right: 30px;
+      width: 470px;
+      height: 70px;
+    }
+  }
+
+  &__send {
+    width: 50px;
```

Comment:
Use a consistent order for CSS properties. You can use alphabetical order or group them by type (e.g., positioning, box model, typography, etc.)